.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,45 @@
+name: Pre-release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  logging:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Print event metadata
+        run: |
+          echo "${{toJSON(github.event.commits[0])}}"
+          echo "${{toJSON(github.event.head_commit)}}"
+  pre-release:
+    if: |
+      !contains(github.event.head_commit.message, 'scalameta/dependabot/') &&
+      !contains(github.event.author.email, 'dependabot[bot]@users.noreply.github.com') &&
+      github.event.author.email != 'bot@scalameta.org'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+      - uses: actions/checkout@v2
+      - name: Setup git env
+        run: |
+          git config --global user.email "bot@scalameta.org"
+          git config --global user.name "Scalameta bot"
+      - run: |
+          yarn install
+          yarn version --no-git-tag-version --patch
+      - name: Commit changes
+        run: |
+          git add .
+          git commit -m "Increase patch version for pre-release"
+      - name: Publish pre-release version to Visual Studio Marketplace
+        uses: HaaLeo/publish-vscode-extension@v1
```

Comment:
Do we need the additional action? Can't we use vsce? This isn't an official action so I would be a bit hesitant to use it.

Reply:
It's actively maintained and it's very easy to use hence I don't see any problem with using it